### PR TITLE
Install instuctions added

### PR DESCRIPTION
**Pull request by Sozialarchiv**

### CODE REVIEW

**Comment by kerim1** [comment]
The part about Docker Compose does not make sense as there is no compose file. And you are missing the `IIIF_IMAGE_CONCURRENCY` parameter, which controls the number of threads libvips can use for image processing. By default it takes the number of CPU cores available. Or you specify 0 to make this explicit.

**Comment by Sozialarchiv** [comment]
> The part about Docker Compose does not make sense as there is no compose file. 

I removed it.

>  And you are missing the IIIF_IMAGE_CONCURRENCY parameter, which controls the number of threads libvips can use for image processing. By default it takes the number of CPU cores available. Or you specify 0 to make this explicit.

I added it to the doc.

File: readme.md [hunk 1]
```diff
@@ -17,14 +17,6 @@ E.g. http://localhost:3333/example.jpg/full/!100,100/0/default.jpg
 
 Use the provided Docker Compose or install manually.
```
Comment (kerim1) [inline readme.md]: Missed this part about Docker Compose

**Review by kerim1 — COMMENTED** [review]
Just the one last thing about Docker Compose to be removed and then it can be merged.

**Pull request merged**